apps/web-giddh/src/app/material-ledger/ledger.component.html

```diff
@@ -920,16 +946,36 @@ <h3 class="ledger-main-head">
                                                                 class="select2-parent select2-ledger pl-0 pr-0"
                                                             >
                                                                 <div
+                                                                    *ngIf="!txn.showAccountDropdown"
                                                                     class="w-100"
                                                                     (click)="
-                                                                        txn.showDropdown = true;
-                                                                        showBankLedgerPopup(txn, item)
+                                                                        showBankLedgerPopup(txn, item);
+                                                                        txn.showAccountDropdown = true
                                                                     "
                                                                 >
                                                                     <span
                                                                         [ngClass]="{
-                                                                            'account-placeholder': true,
-                                                                            'account-placeholder-text':
+                                                                            'account-placeholder yes': true,
```

Comment:
Please check class name
```suggestion
                                                                            'account-placeholder-yes': true,
```